docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -267,6 +271,46 @@ For IBM Cloud account access management, consult the [IBM Cloud Identity and Acc
 
 For Docker EE cluster access management, use the [UCP Access Control documentation](/datacenter/ucp/2.2/guides/access-control/).
 
+## Secure public network access to swarms
+
+By default, Docker EE for IBM Cloud uses [IBM Cloud Security Groups](https://console.bluemix.net/docs/infrastructure/security-groups/sg_index.html#getting-started-with-security-groups) to control access to your clusters by setting rules for incoming and outgoing traffic. You need to have [permissions in your IBM Cloud infrastructure account](faqs.md#what-ibm-cloud-infrastructure-permissions-do-i-need) to manage security groups so that you can provision clusters with security groups. There are two security groups: one for the DTR and worker nodes, and one for the manager nodes.
+
+The security group for DTR and worker nodes blocks all traffic on the public network. DTR and service load balancers communicate with these nodes through the cluster's private network. For public access to services that are running on worker nodes, [use the service load balancer](load-balancer.md#service-load-balancer).
+
+The security group for the manager nodes allows public network traffic to the manager nodes only through port 56422. To access the cluster's manager nodes:
+
+1. Log in to the IBM Cloud CLI. If you have a federated ID, use the `--sso` option.
```

Comment:
I don't remember if I told you this trick before, but if you add an extra space between single-numbered list numbers and the text of the bullet list, then everything inside the list item lines up nicer and things still show correctly in the output. Consider:

```markdown
1. How you have it, if I put a code block I have to use three spaces instead of four (so not on a tab stop) to line up things in this list item.
   
   This is a second paragraph but it could be a code block.

2.  This lines up nicer.
    
    This paragraph starts at a tab stop.
```

Reply:
Thanks for the tip! 

I'll capture that and play around with it. I might have Atom set up weird, because if I start a new line, it makes it the second bullet automatically ( `2.`) and I have to backspace out the indentation anyways.